### ISSUE TITLE
Add privet to connector

### ISSUE THREAD

**Issue by jacobmarble** [google/cups-connector#6]
Add privet capability for local discovery and printing.

https://developers.google.com/cloud-print/docs/privet

**Comment by jacobmarble** [comment]
Privet should support offline printing for users with special privacy concerns. Offline printing means that a print job never leaves the physical site:

1) job originates on the user's workstation or mobile device
2) workstation passes job to the connector
3) connector passes job to CUPS with processing requirements (some are normally handled by Cloud Print in Google's data centers)
4) CUPS sends job to the printer

**Comment by jacobmarble** [comment]
Quick clarification. #35 will add local discovery/printing as an option. Before this issue is closed, another pull request will make cloud discovery/printing optional, so that **either** cloud, **or** local **or both** can be used to share printers. This is important for low-bandwidth circumstances like #40 and for privacy-sensitive content that should have no chance of leaving the local subnet.

**Comment by mceachen** [comment]
I rebuilt my connector, but mast can't read prior configuration files. I re-created the config (which meant the printers needed to be re-shared), but when I changed "local_printing_enable" to true and restarted `connector`, the printer was then visible from a chromebook on the same LAN (yay), but after clicking "print" in the dialog, an error popped up and nothing printed (boo).

What debug logging would be useful?
